frontend/src/js/snack-message/reducer.ts

```diff
@@ -4,12 +4,19 @@ import { Action } from "../app/actions";
 
 import { resetMessage, setMessage } from "./actions";
 
+export enum SnackMessageTypeT {
```

Comment:
Ending types with a`T` is something we're trying to slowly move away from, I'd suggest calling it `SnackMessageType`